bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(
+            filename, file, config_file,
+            use_spaces: bool = False):
```

Comment:
you have an indentation problem here

might be better if you coala over your project 😉

Reply:
then why gitmate didnt gave indentation error?

Reply:
Btw the indentations according to PEP8 stds.

Reply:
However I would follow other bears in this case, and do
```python
    def create_arguments(filename, file, config_file,
                         use_spaces: bool = False):
```

Reply:
(Note that the spaces around `=` should also be omitted)